app/grandchallenge/challenges/tasks.py

```diff
@@ -1,37 +1,55 @@
 from celery import shared_task
 from django.core.mail import mail_managers
+from django.db.models import Count, Max
 from requests import exceptions, get
 
 from grandchallenge.challenges.models import Challenge, ExternalChallenge
 from grandchallenge.evaluation.models import Evaluation
+from grandchallenge.profiles.admin import User
 from grandchallenge.subdomains.utils import reverse
 
 
 @shared_task
 def update_challenge_results_cache():
-    for c in Challenge.objects.all():
-        kwargs = {
-            "cached_num_participants": c.participants_group.user_set.all().count()
-        }
-
-        challenge_results = Evaluation.objects.filter(
-            submission__phase__challenge=c, published=True
-        ).order_by("-created")
+    challenges = Challenge.objects.all()
+    evaluation_info = (
+        Evaluation.objects.filter(rank__gt=0, published=True)
```

Comment:
```suggestion
        Evaluation.objects.filter(published=True)
```

Realised that rank is not required as we want all of the published results.